azure-ci.yml

```diff
@@ -9,13 +9,13 @@ jobs:
   # Job names can contain alphanumeric characters and '_', cannot start with a number
   - job: Cypress_e2e_tests
     pool:
-      vmImage: 'ubuntu-16.04'
+      vmImage: 'ubuntu-latest'
```

Comment:
Per https://github.com/actions/runner-images/issues/3287:
> Traditional 5-years support of Ubuntu 16.04 by Canonical [ends in April, 2021](https://ubuntu.com/16-04). To keep our environment updated and secured, we will remove Ubuntu 16.04 from GitHub on September 20, 2021, and from Azure DevOps on October 18, 2021.

This failed to run: https://cypress-io.visualstudio.com/cypress-example-kitchensink/_build/results?buildId=1642&view=logs&j=d2f18934-c168-59e0-ad20-3282681c6565